docs/framework/wcf/bindings-overview.md

```diff
@@ -33,6 +33,8 @@ Bindings are objects that are used to specify the communication details that are
 -   <xref:System.ServiceModel.NetNamedPipeBinding>: Uses the [!INCLUDE[dnprdnshort](../../../includes/dnprdnshort-md.md)] to connect to other WCF endpoints on the same machine.  
   
 -   <xref:System.ServiceModel.NetMsmqBinding>: Uses the [!INCLUDE[dnprdnshort](../../../includes/dnprdnshort-md.md)] to create queued message connections with other WCF endpoints.  
+
+- NetTcpBinding: This binding offers higher performance than HTTP bindings and is ideal for use in a local network.
```

Comment:
This should also use the xref syntax.  For example:

`<xref:System.ServiceModel.NetTcpBinding>`

That way, the build produces a link to the API reference.